lib/apia/open_api/objects/parameters.rb

```diff
@@ -87,14 +103,31 @@ def generate_argument_set_params
             next if child_arg.type.argument_set?
 
             param = {
-              name: "#{@argument.name}[#{child_arg.name}]",
-              in: "query",
-              schema: generate_scalar_schema(child_arg)
+              in: "query"
             }
+
             description = []
             description << formatted_description(@argument.description) if @argument.description.present?
             description << formatted_description(child_arg.description) if child_arg.description.present?
-            description << "All '#{@argument.name}[]' params are mutually exclusive, only one can be provided."
+
+            if @argument.type.id.end_with?("Lookup")
+
+              description << "\n\nAll '#{@argument.name}[]' params are mutually exclusive, only one can be provided."
```

Comment:
This will add the preceding line breaks even if it's the only description, which may render a bit weird. Probably best to make the line breaks conditional on there being some existing descriptions, eg:

```suggestion
              description << "#{description.any? "\n\n" : ''}All '#{@argument.name}[]' params are mutually exclusive, only one can be provided."
```

(or something more readable but you get the idea 😅)